### ISSUE TITLE
Gradient Descent description, incorrect?

### ISSUE THREAD

**Issue by AeroGeekDean** [udacity/sdc-issue-reports#292]
In MiniFlow lesson, part 13 "Gradient Descent Part 1", the ball analogy description indicates that:

"In this case, the height of the point represents the difference between the current output of the network and the correct output given these INPUTS" [emphasis mine]

And the rest of description continues to describe that each new dimension of the plane as a representation of an INPUT.

My question:

Shouldn't it be "parameters" or "weights", instead of "inputs"??

As I understand it:
Inputs are the set of features (X vector) that goes into the model to predict its corresponding outcome (y label value). While parameters (i.e.: weights & biases) are tweaked/optimized by the learning process. With the goal of finding the set of parameters that will minimize the model error (cost). Thus the gradient is the sensitivity of the cost to each parameters, evaluated at their current values.

If I read the published course material correctly, then I think the usage of "INPUT" here is incorrect. Or if "INPUT" is meant to be parameters in this case, then this usage could lead to student confusion.

**Comment by domluna** [comment]
@AeroGeekDean you're right this should be "weights" and "parameters". The location of the ball in the space is based on the parameters, which is what's tweaked during the course of learning. I'll make an update.
